src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.ts

```diff
@@ -70,8 +70,7 @@ export class LogsComponent implements OnInit, OnDestroy {
 
   abstractFilters(): any {
     const priority = this.priority;
-    const key = this.search.toLowerCase().replace(/,/g, '');
-
+    const key = this.search.toLowerCase();
```

Comment:
TBH, I don't know why it was implemented this way in the first place. Suppose  I want to search `30, 40` in logs, so as per previous implementation `'30, 40'` is transformed to `'30 40'` and then checked for exact string match in the cluster logs. By doing so it will find no match.

The ideal way in MHO is to split `'30, 40'` into array of strings `['30', '40']` and find flogs which contain all the string the array. Basically keyword input can contain comma separated strings to be search in the logs.

Reply:
makes sense.